Description: fix an error that made synopsis invisible;
 fix hyphen/minus usage.
Author: Ilya Barygin <barygin@gmail.com>

diff -Nur -x '*.orig' -x '*~' emerald-0.7.2/doc/emerald.1 emerald-0.7.2.new/doc/emerald.1
--- emerald-0.7.2/doc/emerald.1	2010-02-18 23:17:11.000000000 +0300
+++ emerald-0.7.2.new/doc/emerald.1	2010-02-18 23:17:54.000000000 +0300
@@ -2,21 +2,21 @@
 .SH NAME
 emerald - The Emerald Windows Decorator
 .SH SYNOPSIS
-.Bemerald [--replace] [--help] [--version]
+.B emerald [\-\-replace] [\-\-help] [\-\-version]
 .SH DESCRIPTION
 .I emerald
 emerald is the windows decorator of the beryl project.
 \.
 .SH OPTIONS
 .TP 5
-.RI --replace
+.RI \-\-replace
 Replaces a current running decorator
 .TP 5
-.RI --version
+.RI \-\-version
 outputs the current version of
 .I emerald
 .TP 5
---help
+\-\-help
 short usage statement about
 .I emerald
 .SH SIGNALS
